<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historyWidget</class>
 <widget class="QWidget" name="historyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1,1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>History</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="setWallPaper">
       <property name="text">
        <string>Set Wallpaper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delWallPaper">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importBtn">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Playlist</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutPlaylist"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addPlaylistBtn">
       <property name="text">
        <string>Add Playlist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePlaylist">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="loopBox">
       <property name="text">
        <string>Loop play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moreSetting">
       <property name="text">
        <string>Loop playback settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
